xarray/core/variable.py

```diff
@@ -1049,6 +1064,12 @@ def chunk(self, chunks={}, name=None, lock=False):
             )
             chunks = {}
 
+        if isinstance(chunks, (numbers.Number, str, int, tuple, list)):
+            pass  # dask.array.from_array can handle these directly
+        else:
+            chunks = None if chunks == {} else chunks
```

Comment:
```suggestion
            chunks = chunks or None
```
?

Reply:
Actually I think a better fix would be to change line 269 in `either_dict_or_kwargs` 
from
```python
if pos_kwargs is None:
```
to 
```python
if pos_kwargs is None or pos_kwargs == {}:
```

Reply:
Yes great idea!